tests/responses_callbacks.py

```diff
@@ -46,6 +59,10 @@ def messages_callback(request):
         resp_body['token'] = 'invalid'
         resp_body['status'] = 0
         resp_body['errors'] = ['application token is invalid']
+    elif qs.get('attachment', None) and qs.get('attachment', None) != TEST_IMAGE_BYTES:
+        resp_body['attachment'] = 'wrong image'  # todo better error message like real Pushover does
```

Comment:
These are not the real error messages. I need to investigate the Pushover API to see what it returns in various cases.